### ISSUE TITLE
Block editor adds dialog to insert assets

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#23978]
### Parent Issue
#21988 

### User Story

As a user, I want to be able to focus to insert assets to dotCMS easily.

By assets, we mean:

1. Images
2. Videos
3. Contentlets

### Acceptance Criteria

- [ ] When user select images, video or contentlets we show the prime dialog with the asset search
- [ ] When the user click on any of the asset, we insert it to the block editor and close the dialog
- [ ] User will not see the tippy for asset search anymore

### Proposed Objective

Core Features

### Proposed Priority

Priority 3 - Average

### Assumptions & Initiation Needs

In short, instead of using tippy to show the dialog to search and insert assets, we're going to use primeng dialog.
